QWeb/internal/actions.py

```diff
@@ -204,19 +204,27 @@ def right_click(element: WebElement) -> None:
     ac.context_click(element).perform()
 
 
-def mouse_down(element: WebElement) -> None:
+def mouse_down(element: Optional[WebElement] = None) -> None:
     driver = browser.get_current_browser()
     ac = ActionChains(driver)
-    ac.click_and_hold(element).perform()
+    if element:
+        ac.click_and_hold(element)
+    else:
+        ac.click_and_hold()
+    ac.perform()
 
 
-def mouse_up(element: WebElement) -> None:
+def mouse_up(element: Optional[WebElement] = None) -> None:
     driver = browser.get_current_browser()
     ac = ActionChains(driver)
-    ac.release(element).perform()
+    if element:
+        ac.release(element)
+    else:
+        ac.release()
```

Comment:
if-else is not necessary here

`ac.click_and_hold(None)` is the equivalent to `ac.click_and_hold()`

If there's a need for differing error messages for `WebElement` / `None` in the future then keeping the if-else could be beneficial

Reply:
Fixed with a new commmit